gcp/README.md

```diff
@@ -46,6 +46,18 @@ Initial instructions based on [exekube's Getting Started](https://exekube.github
    * @mrtyler requested a quota bump to 100 Projects.
       * He only authorized his own email for now, to see what it did. But it's possible other Ops team members will need to go through this step.
 
+## One-time CI Setup
```

Comment:
Don't we have anything to manage the CI runner box? (not a big fan of any manual steps :))

Reply:
Indeed! We have https://github.com/gpii-ops/ansible-gpii-ci-worker.

I share your preference for automation. There are a few reasons I have this as a manual step:
* It was simple :). Also it was a record of what I had done to facilitate my manual testing while I figured things out.
* Laziness :). Installing credentials on the CI machine is currently a rare event and there are other things to do.
* I forgot that I had already solved a similar problem (docker hub creds) with ansible-gpii-ci-worker.
* I knew that some details about auth/creds would change with Alfredo's work in #60, so I postponed a more robust solution.
* This step currently has a manual component regardless, since a human must use their credentials to obtain `owner.json`.

All of that said, you are certainly wise to raise the question! I was going to respond by adding `owner.json` to the ansible vault and deploying it automatically with ansible-gpii-ci-worker.

However, now that I'm pursuing your suggestion of using Volumes instead of Bind Mounts, I'm not sure exactly how I'm going to provide `owner.json`. Let's talk about it in my new PR, #92.